develop-guide/writeblitz/writeblitz.rst

```diff
@@ -570,28 +573,29 @@ Action ``compare`` allows you to verify the values of the saved variables. Below
         - " %VARIABLES{bootflash} >= 290000 or '%VARIABLES{bios}' == '07.33'"
 
 
-
 .. note::
 
-    The starting message of an step in etrade logs, is predefined. However, that value can be changed.
-    In your action you can specify your custom message like the example below and change the starting action log message.
+    The starting message of a Step can be modified by specifying a custom message like the example below. This can be applied
+    to all the actions supported in Blitz.
 
 .. code-block:: YAML
 
     - execute:
         command: show version
         device: PE1
-        custom_step_message: This is a test to see if a custom_step_message would be applied
+        custom_start_step_message: This is a test to see if a custom_start_step_message would be applied
```

Comment:
Not a good message for a doc; if you are writing doc it better work :D 

        custom_start_step_message: My own message instead of the default one

Reply:
Done